### PR TITLE
draft changes in contributing-md

### PR DESCRIPTION
**Pull request by jcolomb**
I think some links should be modified at the end, I do not know about shell-novice and co

Please delete this line and the text below before submitting your contribution.

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.

---

### CODE REVIEW

File: inst/templates/contributing-template.txt [hunk 1]
```diff
@@ -42,27 +42,18 @@ and to meet some of our community members.
...
-    which documents the format of our lessons
-    and can be viewed at <https://carpentries.github.io/lesson-example>.
-
+    - not applicable
 3.  If you wish to change the template used for workshop websites,
-    please work in <https://github.com/carpentries/workshop-template>.
-    The home page of that repository explains how to set up workshop websites,
-    while the extra pages in <https://carpentries.github.io/workshop-template>
-    provide more background on our design choices.
+    please refer to [template-doc].
```
Comment (zkamvar) [inline inst/templates/contributing-template.txt:55]:
```suggestion
    please refer to [The Workbench documentation][template-doc].
```

Comment (zkamvar) [inline inst/templates/contributing-template.txt:55]:
```suggestion
    please refer to [The Workbench documentation][template-doc].
```

File: inst/templates/contributing-template.txt [hunk 2]
```diff
@@ -106,13 +97,10 @@ you may want to look at
...
-1.  The published copy of the lesson is in the `gh-pages` branch of the repository
-    (so that GitHub will regenerate it automatically).
+1.  The published copy of the lesson is in the `main` branch of the repository,
+    that branch should be protected from direct commits.
     Please create all branches from that,
-    and merge the [master repository][repo]'s `gh-pages` branch into your `gh-pages` branch
-    before starting work.
-    Please do *not* work directly in your `gh-pages` branch,
-    since that will make it difficult for you to work on other contributions.
+    discuss with the maintainer about what branch to use before starting to work.
```
Comment (zkamvar) [inline inst/templates/contributing-template.txt:103]: I believe this entire sentence can be deleted since GitHub flow is described below.

**Review by zkamvar — COMMENTED** [review]
This looks good to me. Thank you for starting this!

Could you add your name to the DESCRIPTION as a contributor? (here is a [guide for the format](https://r-pkgs.org/description.html#author-who-are-you).

I am currently trying waiting for #405 to pass and then I will merge this and fix any tests that may need fixing.

**Review by jcolomb — COMMENTED** [review]
I think it would be good to go through the list of links in the end, change things that should be modified with `FIXME` and see that other links are not related to old template

**Review by jcolomb — COMMENTED** [review]
add modify

**Comment by jcolomb** [comment]
@zkamvar now that #405 is merged, can we merge or do you want to discuss keeping the section about th

**Comment by zkamvar** [comment]
Sorry about the lack of communication on my end; Aside from the chaos of the last week, I had an emergency situation yesterday that took me away from my desk and could not address this. Honestly, it never usually takes me this long to address issues, especially something as straightforward as modifying a template file.

> can we merge or do you want to discuss keeping the section about the branch being head

For me, it is valuable to keep an acknowledgement of how we used to do things so that folks who transition to using The Workbench from styles know the difference, but it should be an aside. My reasoning is that this document should accurately describe our current practices to allow people to quickly and confidently make a contribution. 

The discussion of the branch names was a relic from a time when lessons often had a default branch that was different than the generally accepted default branch (and even at that, our lesson repositories were not consistent in this respect), which is now no longer relevant.

**Comment by jcolomb** [comment]
no problem, you are still one of the fastest people around even when you fell you are slow :)

I simplified a bit, taking our remnants from previous workflows.

**Comment by zkamvar** [comment]
Thank you @jcolomb! T

**Pull request merged**